PopPUNK/__main__.py

```diff
@@ -757,10 +798,15 @@ def main():
 #*                             *#
 #*******************************#
 def assign_query(dbFuncs, ref_db, q_files, output, update_db, full_db, distances, microreact, cytoscape,
-                 kmers, sketch_sizes, ignore_length, estimated_length, threads, use_mash, mash, overwrite,
+                 kmers, sketch_sizes, estimated_length, threads, use_mash, mash, overwrite,
                  plot_fit, no_stream, max_a_dist, model_dir, previous_clustering,
                  external_clustering, core_only, accessory_only, phandango, grapetree,
-                 info_csv, rapidnj, perplexity, assign_lineage, existing_scheme, rank_list, use_accessory):
+                 info_csv, rapidnj, perplexity, assign_lineage, existing_scheme, rank_list, use_accessory,
+                 # added extra arguments for constructing sketchlib libraries
+                 reads = False, strand_preserved = False, min_count = 0,
+                 use_exact = False, qc_filter = 'stop', retain_failures = False,
+                 length_sigma = 5, lower_length = None, upper_length = None, prop_n = 0.1,
+                 upper_n = None):
```

Comment:
Probably best not to have the defaults defined twice - I'd remove them here

Reply:
Removed in 258fe53.